gen/plugins/com.salesforce/com.salesforce.plugin.network.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2012-present, salesforce.com, inc.
```

Comment:
That's because of the `split` script. The big `cordova_plugins.js` has `2012-present`, and the script splits them up.